src/components/elements/livechat-link-text.tsx

```diff
@@ -2,22 +2,23 @@ import React from 'react'
 import { Localize } from 'components/localization'
 import { useLivechat } from 'components/hooks/use-livechat'
 import { LinkText } from 'components/elements'
+import { TString } from 'types/generics'
 
 type LiveChatLinkTextProps = {
     text?: TString
     weight?: string
 }
 
 const LiveChatLinkText = ({ text, weight }: LiveChatLinkTextProps) => {
-    const [is_livechat_interactive, LC_API] = useLivechat()
+    const [is_livechat_interactive] = useLivechat()
 
     return (
         <LinkText
             color="red"
             mt="1rem"
             weight={weight}
             onClick={() => {
-                is_livechat_interactive && LC_API.open_chat_window()
+                is_livechat_interactive && window.LC_API.open_chat_window()
```

Comment:
@niloo-fs Why are you directly accessing `LC_API` from the `window` object? 🤔 
The idea behind getting it from the `useLivechat` hook was to hide the complexity and make it easer to change it in the future if needed, Not all the codebase need to know `LC_API` is coming from the `window`.

Reply:
You told me that :))))

Reply:
@niloo-fs Let’s have a call on this whenever you are free 😅